Mathlib/Data/Sym/Card.lean

```diff
@@ -5,8 +5,10 @@ Authors: Yaël Dillies, Bhavik Mehta, Huỳnh Trần Khanh, Stuart Presnell
 -/
 import Mathlib.Algebra.BigOperators.Basic
 import Mathlib.Data.Finset.Sym
+import Mathlib.Data.Finsupp.Multiset
```

Comment:
I'm not sure how much to care about introducing this import. I'm inclined to say this is OK since the only file currently importing `Mathlib.Data.Sym.Card` is `Mathlib.Combinatorics.SimpleGraph.Finite`, and this is only used in the other `SimpleGraph` corner of the library.